edu-services/gradebook-service/impl/src/java/org/sakaiproject/component/gradebook/GradebookExternalAssessmentServiceImpl.java

```diff
@@ -989,4 +990,14 @@ public boolean isCategoriesEnabled(String gradebookUid) {
 		final Gradebook gradebook = getGradebook(gradebookUid);
 		return gradebook.getCategory_type() != GradebookService.CATEGORY_TYPE_NO_CATEGORY;
 	}
+
+	@Override
+	public OptionalLong getInternalAssessmentID(final String gradebookUUID, final String externalID) throws GradebookNotFoundException, AssessmentNotFoundException {
```

Comment:
Again, Assessment is confusing. I'd rather just use GradebookItem

Reply:
See my above comment.